docs/install/any-kubernetes-cluster.md

```diff
@@ -439,13 +439,14 @@ The following commands install the Knative Eventing component.
 1. Install the [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) (aka CRDs):
 
    ```bash
-   kubectl apply --filename {{< artifact repo="eventing" file="eventing-crds.yaml" >}}
+   kubectl apply  --selector knative.dev/crd-install=true \
```

Comment:
I think we've been preferring the `-crd` and `-core` files rather than using the label filter.

I don't recall why, but @mattmoor or @vaikas may remember what issues we hit.

Reply:
From https://knative.slack.com/archives/C93E33SN8/p1590510270014300 :

> Kubectl doesn’t apply the selector before blowing up on custom resources

https://github.com/knative/test-infra/issues/1858